repository-data/webfiles/src/main/resources/site/less/demo.less

```diff
@@ -1,303 +1,458 @@
+
+.flush {
+  margin:0px;
+}
+.flush--top {
+  margin-top:0px;
+}
+.flush--bottom {
+  margin-bottom:0px;
+}
+.flush--left {
+  margin-left:0px;
+}
+.flush--right {
+  margin-right:0px;
 }
 
-.column-half__left {
-    float: left;
-    clear: left;
-    width: 49%;
+
+.push--top {
+  margin-top:@base-spacing-unit;
+}
+.push-double--top {
+  margin-top:@double-spacing-unit;
+}
+.push-half--top {
+  margin-top:@half-spacing-unit;
 }
 
-.column-half__right {
-    float: right;
-    clear: right;
-    width: 49%;
+.push--bottom {
+  margin-bottom:@base-spacing-unit;
+}
+.push-double--bottom {
+  margin-bottom:@double-spacing-unit;
+}
+.push-half--bottom {
+  margin-bottom:@half-spacing-unit;
 }
 
 
-/*
-<a class="logo logo-nhs-digital">
-    text that will not be displayed
-</a>
-*/
-.logo {
-    display: block;
-    height: 100px;
-    width: 100px;
-    background-repeat: no-repeat;
-    text-indent: -9999em;
+
+
+
+
+/* HEADER */
+/* ########################################################## */
+
+.header {
+  width:@max-width;
+  margin:0px auto;
+  padding-top:20px;
+  padding-bottom:3px;
 }
-.logo-nhs-digital {
-    background-image: url(img/logo.png);
-    height: 85px;
-    width: 100px;
+
+.header__col1 {
+  display:inline-block;
+  width:20%;
+}
+.header__col2 {
+  display:inline-block;
+  width:80%;
+  vertical-align: bottom;
+  text-align: right;
 }
 
 
+.header__logo {
+  width:80px;
+}
 
+.header__nav__list {
+  margin: 0px;
+  padding: 0px;
+}
 
-input {
-    box-sizing: border-box;
+.header__nav__list__item {
+  display:inline-block;
+  a {
     display: inline-block;
-    line-height: normal;
-    font-size: 1em;
-    margin: 0;
-    padding: 0;
-}
-.search-input {
-    border: 1px solid #CCC;
-    box-sizing: border-box;
-    border-color: #004075;
-    padding: .5em .5em .5em 1em;
-    -webkit-appearance: none;
-    cursor: pointer;
-    width: 400px;
-}
-
-.search-submit {
-    background: #005eb8 url(img/search-icon.png) no-repeat center;
-    border: none;
-    padding: .5em 0 .5em 0;
-    color: white;
-    cursor: pointer;
-    text-indent: -9999em;
-    transition: opacity .3s;
-    width: 3em;
-}
-
-/* Pagination styling from Bootstrap css. To replace when styling is defined for site */
-.pagination {
-    display: inline-block;
-    padding-left: 0;
-    margin: 20px 0;
-    border-radius: 4px;
+    background-image: url(img/icon-chevron-down.svg);
+    background-repeat: no-repeat;
+    background-position: 100% 58%;
+    color: #005eb8;
+    background-size: 20px 20px;
+    padding: 10px 30px 10px 4px;
+    margin-right: 27px;
+    font-size:16px;
+    text-decoration: none;
+  }
 }
 
-.pagination li {
-    display: inline;
-}
 
-.pagination-container {
-    margin-bottom: 20px;
-    overflow: hidden;
-    text-align: center;
+
+
+/* SEARCH */
+/* ########################################################## */
+
+.search {
+  background-color: @color-secondary;
```

Comment:
As discussed, can we add some qualifier to this and derived classes, renaming it to something like 'common-search'?